docs/poller.md

```diff
@@ -47,22 +51,22 @@ The poller creates a log file called /tmp/sq-poller.log. You can look at the fil
 If you're using SSH to connect to the devices (only Arista EOS uses the REST API), the supported models for specifying login credentials are:
 
 * Put the password in the inventory file and ensure no one can read it
-* Use --ask-pass to then be prompted for a password
-* Use an environment var to store the password and pass the name of the env var via --envpass
+* Use `--ask-pass` to then be prompted for a password
+* Use an environment var to store the password and pass the name of the env var via `--envpass`
 * Use keyfile
-* Use passphrase protected keyfile  (use --passphrase)
+* Use passphrase protected keyfile  (use `--passphrase`)
 * Use ssh-config (not well-tested)
```

Comment:
While you're fixing this, could you please help me by removing the "(not well-tested)" line? Its been tested now.